DESCRIPTION

```diff
@@ -1,25 +1,25 @@
 Package: MicroRaman
 Type: Package
-Title: Phenotyping of microbial Raman data
-Version: 0.1.1
+Title: Phenotyping of microbial Raman spectroscopy data
+Version: 0.2
 Authors@R: c(person("Frederiek-Maarten", "Kerckhof", email = "FrederiekMaarten.Kerckhof@UGent.be", role = c("aut", "cre")),
-             person("Benjamin", "Buysschaert", email = "Benjamin.Buysschaert@UGent.be", role = "ctb"),
-             person("Ruben", "Props", email = "Ruben.Props@UGent.be", role = "aut"),
-             person("Cristina", "Garcia-Timermans", email = "Cristina.GarciaTimermans@UGent.be", role = "aut"),
-             person("Dmitry", "Khalenkow", email = "Dmitry.Khalenkow@Ugent.be", role = "ctb"))
```

Comment:
Dmitry was added here due to collab on the original Garcia Timermans et al. publication, but indeed has not contributed to the package